docs/rst/installation/linux_installation.rst

```diff
@@ -2,3 +2,325 @@
+
+            mkdir -p ~/Fast-DDS-QoS-Profiles-Manager/build/lib
+            cd ~/Fast-DDS-QoS-Profiles-Manager/build/lib
+            cmake ../../src/Fast-DDS-QoS-Profiles-Manager/lib -DCMAKE_INSTALL_PREFIX=../../install/lib
+            cmake --build . --target install
+
+        .. note::
+
+            If test compilation is needed, the enable test build flag `EPROSIMA_BUILD_TESTS` should be included:
+
+            .. code-block:: bash
+
+                cmake ../../src/Fast-DDS-QoS-Profiles-Manager/lib -DCMAKE_INSTALL_PREFIX=../../install/lib -DEPROSIMA_BUILD_TESTS=ON
+
+.. _cli_installation_sl:
+
+Fast DDS QoS Profiles Manager CLI
+---------------------------------
+
+This section describes the instructions for installing *eProsima Fast DDS QoS Profiles Manager CLI* in a Linux
+environment from sources.
+
+The *eProsima Fast DDS QoS Profiles Manager library* is required to use the *CLI*, so make sure all steps described in
+:ref:`its installation section <library_api_installation_sl>` were reproduced.
+Despite that, :ref:`cli_dependencies_sl` detailed below need to be met.
+
+.. _cli_dependencies_sl:
+
+Dependencies
+^^^^^^^^^^^^
+
+*eProsima Fast DDS QoS Profiles Manager CLI* depends on `docopt CLI description language <http://docopt.org/>`_.
+In Ubuntu, the dependency can be installed running:
+
+.. code-block:: bash
+
+    sudo apt install libdocopt-dev
+
+Optionally, `GTest <https://github.com/google/googletest>`_ dependency could be installed to run the test suite.
+
+.. _cli_building_sl:
+
+Build
+^^^^^
+
+.. tabs::
+
+    .. tab:: Building with colcon
+
+        *eProsima Fast DDS QoS Profiles Manager CLI* is build using `colcon <https://colcon.readthedocs.io/en/released/>`_
+        by running:
+
+        .. code-block:: bash
+
+            cd ~/Fast-DDS-QoS-Profiles-Manager
+            colcon build --packages-select fastdds_qos_profiles_manager_lib fastdds_qos_profiles_manager_cli
+
+        .. note::
+
+            As happened in the *library* build process, if test compilation is needed, the enable test build flag
+            `EPROSIMA_BUILD_TESTS` should be included too:
+
+            .. code-block:: bash
+
+                colcon build --packages-select fastdds_qos_profiles_manager_lib fastdds_qos_profiles_manager_cli --cmake-args -DEPROSIMA_BUILD_TESTS=ON
+
+        The ``fastddsqosprof`` executable file is generated in the installation path.
+        The following setup call must be run in order to set the *CLI* command:
+
+        .. code-block:: bash
+
+            source ~/Fast-DDS-QoS-Profiles-Manager/install/setup.bash
+
+    .. tab:: Building with CMake
+
+        *eProsima Fast DDS QoS Profiles Manager CLI* is build using `CMake <https://cmake.org/>`_ by running:
+
+        .. code-block:: bash
+
+            mkdir -p ~/Fast-DDS-QoS-Profiles-Manager/build/cli
+            cd ~/Fast-DDS-QoS-Profiles-Manager/build/cli
+            CMAKE_PREFIX_PATH+=~/Fast-DDS-QoS-Profiles-Manager/install/lib
```

Comment:
```suggestion
            CMAKE_PREFIX_PATH+=:~/Fast-DDS-QoS-Profiles-Manager/install/lib
```

This instruction only works if the environment variable has been exported (declared) previously. I suggest using:

```suggestion
            export CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}:~/Fast-DDS-QoS-Profiles-Manager/install/lib
```

This way the variable is ensured to be declared.